### PR TITLE
WF A11y documentation

### PR DESCRIPTION
**Pull request by dmetzgar**

### CODE REVIEW

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+The Windows Workflow Foundation (WF) workflow designer is improving how it works with accessibility technologies. These improvements include the following changes:
+- Tab order changed to left->right & top->bottom in some controls
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: Tab order changed... --> The tab order is changed to left to right and to top to bottom in some controls:

Also, each sentence should end in a period.

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  - Initialize correlation window for setting correlation data for the InitializeCorrelation activity
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: Initialize correlation window for --> The initialize correlation window, for
InitializeCorrelation --> `T:System.ServiceModel.Activities.InitializeCorrelation` 

Also, is this a fairly complete list, or just a few "for example" items. If the latter, can it be expanded to be more comprehensive to have approximately the same level of coverage as [WinForms accessibility improvements](https://github.com/Microsoft/dotnet/blob/master/Documentation/compatibility/winforms-accessibility-changes-471.md)?

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  - Content definition window for Receive, Send, SendReply, and ReceiveReply activities
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: link to classes for each activity (e.g., `T:System.ServiceModel.Activities.Receive`)?

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+- More functions available via keyboard
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: via --> via the

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+  - When editing the properties of an activity, fixed an issue where property groups could not be collapsed by keyboard the first time they are focused
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: fixed an issue where property groups could not be collapsed --> property groups can be collapsed

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+  - Warning icons are now accessible by keyboard
+  - More Properties button in properties window is no accessible by keyboard
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: The More Properties button in the Properties window is now accessible by keyboard.

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+- Users who choose High Contrast themes will see many improvements in the visibility of the workflow designer and its controls
+
+
+- [x] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+If you have an application with a re-hosted workflow designer, your application can benefit from these changes by performing either of these actions:
+- Recompile your application targeting .NET Framework 4.7.1. These accessibility changes are enabled by default.
+- Opt out of the legacy accessibility behaviors by adding the following [AppContext switch](https://docs.microsoft.com/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element) to the `<runtime>` section of the app.config file and set it to `false`, as the following example shows.
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: Opt out... --> If you application targets the .NET Framework 4.7 or earlier but is running on the .NET Framework 4.7.1, you can opt out of these legacy accessibility behaviors...

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+### Change Description
+The Windows Workflow Foundation (WF) workflow designer is improving how it works with accessibility technologies. These improvements include the following changes:
+- Tab order changed to left->right & top->bottom in some controls
+  - Initialize correlation window for setting correlation data for the InitializeCorrelation activity
+  - Content definition window for Receive, Send, SendReply, and ReceiveReply activities
+- More functions available via keyboard
+  - When editing the properties of an activity, fixed an issue where property groups could not be collapsed by keyboard the first time they are focused
+  - Warning icons are now accessible by keyboard
+  - More Properties button in properties window is no accessible by keyboard
+  - Keyboard users now can access the header items in the arguments and variables panes of the workflow designer
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: arguments --> Arguments
variables --> Variables
workflow designer --> Workflow Designer

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+- Improved visibility of focused items such as when
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: focused items--> items with the focus,

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+  - Adding rows to data grids used by the workflow designer and activity designers
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: workflow designer --> Workflow Designer

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+  - Tabbing through fields in the ReceiveReply and SendReply activities
+  - Setting default values for variables or arguments
+- Users of screen readers will see several improvements where the screen reader can now correctly recognize
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: Simply "Screen readers can now correctly recognize:"

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 12]
```diff
@@ -0,0 +1,123 @@
...
+  - Breadcrumbs in workflow designer
+- Users who choose High Contrast themes will see many improvements in the visibility of the workflow designer and its controls
+
+
+- [x] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+If you have an application with a re-hosted workflow designer, your application can benefit from these changes by performing either of these actions:
+- Recompile your application targeting .NET Framework 4.7.1. These accessibility changes are enabled by default.
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: targeting --> to target the

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+  - Warning icons are now accessible by keyboard
+  - More Properties button in properties window is no accessible by keyboard
+  - Keyboard users now can access the header items in the arguments and variables panes of the workflow designer
+- Improved visibility of focused items such as when
+  - Adding rows to data grids used by the workflow designer and activity designers
+  - Tabbing through fields in the ReceiveReply and SendReply activities
+  - Setting default values for variables or arguments
+- Users of screen readers will see several improvements where the screen reader can now correctly recognize
+  - Breakpoints set in the workflow designer
+  - FlowSwitch, FlowDecision, and CorrelationScope activities
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: FlowSwitch --> The FlowSwitch
what is the improvement? 
Also, does it make sense to link to the corresponding classes for FlowSwitch, etc.?

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 14]
```diff
@@ -0,0 +1,123 @@
...
+  - Contents of the Receive activity
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: Contents of --> The contents of

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 15]
```diff
@@ -0,0 +1,123 @@
...
+  - Target type for the InvokeMethod activity
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: Target type --> The target type 
InvokeActivity as a link?

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 16]
```diff
@@ -0,0 +1,123 @@
...
+  - Exception combobox and Finally section in the TryCatch activity
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: Exception combobox and Finally section --> The Exception combo box and the Finally section
TryCatch as a link?

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 17]
```diff
@@ -0,0 +1,123 @@
...
+  - Message type combobox, splitter in Add Correlation Initializers window, Content Definition window, and CorrelatesOn Defintion window in messaging activities
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: The Message Type combo box, the spliiter in the Add Correlation Initializers window, the Content Definition window, and the Correlates on Definition window in messaging activities.

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 18]
```diff
@@ -0,0 +1,123 @@
...
+  - State machine transitions and transitions destinations
+  - Annotations and connectors on FlowDecision activities
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: FlowDecision as a link?

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 19]
```diff
@@ -0,0 +1,123 @@
...
+  - Context (right-click) menus for activities
+  - Property value editors, clear search button, By Category and Alphabetical sort buttons, and expression editing dialog in the properties grid
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: clear search --> the Clear Search (capitalized?)
By Category --> the By Category
expression editing dialog --> the Expression Editing dialog (capitalized?)

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 20]
```diff
@@ -0,0 +1,123 @@
...
+  - Zoom percentage in the workflow designer
+  - Separator in parallel and pick activities
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: Separator --> The separator

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 21]
```diff
@@ -0,0 +1,123 @@
...
+  - InvokeDelegate activity
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: InvokeDelegate --> The InvokeDelegate 
InvokeDelegate as a link?

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 22]
```diff
@@ -0,0 +1,123 @@
...
+  - Select Types window for Dictionary activities
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: Select Types --> The Select Types
Dictionary as a link

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 23]
```diff
@@ -0,0 +1,123 @@
...
+  - Browse and Select .NET Type window
+  - Breadcrumbs in workflow designer
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: the Workflow Designer.

File: Documentation/compatibility/workflow-designer-accessibility.md [hunk 24]
```diff
@@ -0,0 +1,123 @@
...
+- Users who choose High Contrast themes will see many improvements in the visibility of the workflow designer and its controls
```
Comment (rpetrusha) [inline Documentation/compatibility/workflow-designer-accessibility.md]: Can you expand this a bit: what improvements?

**Comment by dmetzgar** [comment]
Thanks @rpetrusha.  Updated.

**Pull request merged**